en/peripherals/gripper.md

```diff
@@ -0,0 +1,107 @@
+# Gripper Peripherals
+
+Grippers are mechanical devices that can be integrated with an unmanned vehicle to hold (grip) and release payloads.
+
+PX4 allows grippers to triggered automatically in [Payload Delivery Missions](../flying/package_delivery_mission.md) or manually using a Joystick.
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+:::note
+A gripper can also be configured as a [generic RC or MAVLink actuator](../payloads/README.md#actuator-control-with-rc).
+This allows it to be used manually via an RC Controller, but not in missions or on a Joystick.
+:::
+
+## Supported Grippers
+
+There are many different gripper mechanisms ("jaws", "fingers", "electromagnets") and interfaces (PWM, CAN, MAVLink, and so on).
+
+PX4 supports grippers that have simple triggers to hold and release, and that use the following interfaces (see linked documents for details):
+
+- [PWM Servo Gripper](gripper_servo.md) - Grippers connected to autopilot PWM outputs
+- **MAVLink Gripper** (Untested) - Grippers that support the [MAV_CMD_DO_GRIPPER](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_GRIPPER) MAVLink command.
+
+:::note
```

Comment:
How does Roboclaw interface work?

Reply:
For now it just takes in `actuator_controls_0` topic, and directly maps the throttle and yaw output of the flight controller into the drive & turn commands of the Roboclaw. So I am not sure if it would work so well now, and it probably needs to be refactored to use Control allocation Actuators architecture, right @bkueng ?

Reply:
What I mean is "does it have a PWM interface"? How is it connected? I'd like to say something like "Roboclaw is not yet supported because XXX bus is not integrated with YYYY"

Reply:
@junwoo091400 I have updated the whole of the gripper and gripper servo docs. For those docs only question is this one ^^^